modules/ROOT/pages/release-notes.adoc

```diff
@@ -3,55 +3,54 @@
 
 [NOTE]
 ====
-The 2022.2 is in development.
+The 2022.2 is in qualification.
```

Comment:
In my mind, i think we don't use `qualification` but `beta`.

Reply:
your thought his accepted :) It's indeed more accurate with the version name 👍